README.md

```diff
@@ -3,21 +3,28 @@
 CNWAN Reader watches a service registry for changes and sends events to an
 external handler for processing.
 
-The CNWAN Reader is part of the Cloud Native SD-WAN (CNWAN) project. Please check the [CNWAN documentation](https://github.com/CloudNativeSDWAN/cnwan-docs) for the general project overview and architecture. You can contact the CNWAN team at [cnwan@cisco.com](mailto:cnwan@cisco.com).
+The CNWAN Reader is part of the Cloud Native SD-WAN (CNWAN) project.
+Please check the [CNWAN documentation](https://github.com/CloudNativeSDWAN/cnwan-docs)
+for the general project overview and architecture.
+You can contact the CNWAN team at [cnwan@cisco.com](mailto:cnwan@cisco.com).
 
 ## Table of contents
 
 * [Overview](#overview)  
 * [Supported Service Registries](#supported-service-registries)  
-* [Installing](#installation)
+* [Installation](#installation)
+  * [Go Get](#go-get)
+  * [Clone the Project](#clone-the-project)
+  * [Run as a Docker Container](#run-as-a-docker-container)
 * [Usage](#usage)
   * [CNWAN Adaptor](#cnwan-adaptor)
+  * [Metadata Key](#metadata-key)
   * [Service Directory](#service-directory)
   * [Example](#example)
```

Comment:
```suggestion
  * [Binary Example](#binary-example)
```

Reply:
I have amended and re-pushed this. Thank you for spotting it!